coalib/bearlib/languages/definitions/Tcl.py

```diff
@@ -0,0 +1,12 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Tcl:
+    aliases = 'Tcl/Tk',
+    versions = 8.0, 8.1, 8.2, 8.3, 8.4, 8.5, 8.6
+    extensions = '.tcl',
+    comment_delimiter = '#'
+    string_delimiters = {'"': '"'}
+    multiline_string_delimiters = {'"': '"'}
+    encapsulators = {'(': ')', '[': ']', '{': '}', '[[': ']]'}
```

Comment:
what is the meaning of `[[ ... ]]` ?

Reply:
oops, mistakenly took nested ones as encapsulators, will remove this.